### DISCUSSION TITLE
WindHawk install file is reported by VirusTotal as malicious

### DISCUSSION THREAD

**Discussion by glorifyday** [ramensoftware/windhawk#223] [General]
This is the VirusTotal message.

1/71 security vendor flagged this file as malicious
VBA32 - Suspected Of Trojan.Downloader.gen

What can be done about it?

**Comment by silverbacknet** [comment]
Ignore it and go on with life, like with any other antivirus false-positive? They're just a fact of life, and next week it might have completely different results. You can hound the individual app to fix their junk, but AV systems in general just aren't very good at consistently telling malicious vs benevolent.

**Reply by glorifyday** [reply]
I would like to use it on my work laptop and it would be easier to convince my company to allow it, if the VirusTotal says "all clean".

**Comment by m417z** [comment]
It’s surely a false positive detection. Unfortunately, struggling with false positives is a non-ending hassle.

This problem was discussed many times in the blog comments regarding my tools, for example:  
[https://ramensoftware.com/7-taskbar-tweaker/comment-page-43\#comment-11035](https://ramensoftware.com/7-taskbar-tweaker/comment-page-43#comment-11035)  
[https://ramensoftware.com/contact/comment-page-4\#comment-10292](https://ramensoftware.com/contact/comment-page-4#comment-10292)

See also Nir Sofer's blog post, [Antivirus companies cause a big headache to small developers](https://blog.nirsoft.net/2009/05/17/antivirus-companies-cause-a-big-headache-to-small-developers/), a good write-up about the problem.

You can always help by reporting the detection as a false positive to the relevant antivirus companies.

For VBA32, try contacting feedback@anti-virus.by.

**Reply by glorifyday** [reply]
Thanks a lot for your answer, I'll contact them,

**Reply by GuySiklier** [reply]
If it's false positive why the file suspected name Alibaba it's little suspicious
